### ISSUE TITLE
Error from cpump running the flow test

### ISSUE THREAD

**Issue by benlapETS** [MetPX/sarrac#20]
I got this error running the flow test that i'm not sure if it is relevant investigating, (I ran in debug mode):

In flow_check_errors_logged.txt:
```
1 sr_cpump_xvan_f14_01.log [ERROR: failed to open cache file to load: /local/ben/.cache/sarra/cpump/xvan_f14/recent_files_001.cache
1 sr_cpump_xvan_f15_01.log [ERROR: failed to open cache file to load: /local/ben/.cache/sarra/cpump/xvan_f15/recent_files_001.cache
```

**Comment by petersilva** [comment]
yeah, I get that too...  I only investigated enough to determine that it doesn't do any harm... basically after a --reset, or in a configuration that has never been started before, there will be no cache file.  How to differentiate between a failed startup, and a *new* startup ?
